lxd/db/cluster/open.go

```diff
@@ -211,6 +211,15 @@ INSERT INTO nodes(id, name, address, schema, api_extensions, arch, description)
 				return err
 			}
 
+			// Default admin group
+			var adminGroupStmt strings.Builder
+			adminGroupStmt.WriteString(`INSERT INTO auth_groups (name, description) VALUES ('admin', 'Admin LXD auth group');`)
```

Comment:
"LXD" and "auth group" are redundent given the context

Reply:
changed the description to "Server admin".